test/extended/node/node_e2e/node_utils.go

```diff
@@ -0,0 +1,55 @@
+package node
+
+import (
+	"strings"
+	"time"
+
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+func assertKubeletLogLevel(oc *exutil.CLI) {
```

Comment:
Do you expect that this function will be used in another test?  Maybe it is not needed.  Just put it all in the test.  We can refactor later if we discover that is it reusable.